integration/tests/cook/cli.py

```
@@ -58,6 +58,18 @@ def command():
     return os.environ['COOK_CLI_COMMAND'] if 'COOK_CLI_COMMAND' in os.environ else 'cs'
 
 
+def plugins():
+    """If the COOK_PLUGIN_JSON environment variable is set, returns its value, otherwise empty dict"""
+    if 'COOK_PLUGIN_JSON' in os.environ:
+        try:
+            return json.loads(os.environ['COOK_PLUGIN_JSON'])
+        except ValueError:
+            logging.warn(f"Plugin json '{os.environ['COOK_PLUGIN_JSON']}' cannot be parsed. Ignoring.")
```

Maybe we should `raise` here instead of just logging a warning?